examples/source_examples/aws_athena/using_sqlAlchemy/connector.py

```diff
@@ -0,0 +1,79 @@
+# This is a simple example for how to work with the fivetran_connector_sdk module.
```

Comment:
please be consistent with casing in directory name `using_sql_alchemy`

Reply:
should be SQLAlchemy